ServerCore/Pages/Index.cshtml

```diff
@@ -9,7 +9,7 @@
 <h4>Microsoft Non-Intern Puzzleday 2023</h4>
 <img src="~/images/pd2023/nipd2023logo.png" height="256" width="256" title="Puzzleday 2023 logo" alt="Puzzleday 2023" />
 <p>A one-day rebroadcast for employees and friends that happens every summer. Puzzle solvers of all levels of experience are welcome!</p>
-<a href="https://puzzlehunt.azurewebsites.net/nipd2023/play">Go to the Non-Intern Puzzleday event</a>
```

Comment:
Please undo all these changes to the main landing page.  I assume this is just a find-replace gone amok, but these links are to each event's homepage, where play is just the role (as opposed to author or admin).

Reply:
Thanks for the catch! Fixed